### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by codexmen09** [community/community#48765] [Other Feature Feedback, Questions, & Ideas]
so yesterday i had started some code on open ai and accidentally linked it to github when i saved. when i went back 20 mins later there was collaberators in the code and it had a license on it . it is not in my repository for some reason and i still have the saved file in my  open ai. It had taken me a long time to come up with the idea and plan of execution for what was supposed to come next. i have all the collaborators info in the file. what can i do about this?

**Comment by ettaboyle** [comment]
Hi @Codexmen09, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction. Updating the title to something more descriptive than `help` is best practice too!

Good luck!
